packages/client/ui/react-ui/src/twind.config.ts

```diff
@@ -91,6 +93,12 @@ export default defineConfig({
                 "500": "500ms",
             },
         },
+        fontFamily: {
+            sans: ["Inter", "sans-serif"],
+        },
+    },
+    preflight: {
```

Comment:
we might not wanna preflight the fonts as it may interfere with client apps consuming this lib.

Reply:
Yeah, I wouldn't force a font either, I think we should inherit and make a suggestion at most

Reply:
I removed any font forcing and just use defaults.